ikarus-webapp/src/features/squads/server/publish.js

```diff
@@ -1,4 +1,8 @@
 Meteor.publish('SquadsOnServer', function(serverName){
+  if (!get(Meteor.user(), 'serverId')) {
```

Comment:
add "this.ready()" here to mark this publish ready, even if it does not return anything.